opencti-platform/opencti-front/src/private/components/analysis/reports/ReportLine.tsx

```diff
@@ -160,7 +181,7 @@ const ReportLineComponent = ({
             >
               <ItemMarkings
                 variant="inList"
-                markingDefinitionsEdges={node.objectMarking.edges ?? []}
+                markingDefinitionsEdges={node.objectMarking?.edges ?? []}
```

Comment:
ObjectMarking should probably always be a list

Reply:
ObjectMarking can be null

Reply:
because a report can have no markings (it is the same format for other object with an attribute objectMarking)